docs/conf/eu/2017/news/announcing-presentations-ticket-update.rst

```diff
@@ -0,0 +1,70 @@
+* Raphael Pierzina - `Hi, my name is README! </conf/eu/2017/speakers/#speaker-raphael-pierzina>`_
+* Ruthie BenDor - `Even Naming This Talk Is Hard </conf/eu/2017/speakers/#speaker-ruthie-bendor>`_
+* Stuart Culshaw - `Tech Writers Without Borders: Making the world a better place, one (numbered) step at a time </conf/eu/2017/speakers/#speaker-stuart-culshaw>`_
+* Thomas Parisot - `Writing a book in 2017  </conf/eu/2017/speakers/#speaker-thomas-parisot>`_
+* Tim Rogers - `"You never get a second chance to make a first impression": writing great "getting started" documentation </conf/eu/2017/speakers/#speaker-tim-rogers>`_
+
+Unconference Sessions
+---------------------
+
+* Ed Stenson - `One-on-One 101 </conf/eu/2017/speakers/#speaker-ed-stenson>`_
+* Floriana Pagano - `Managing community-driven documentation - how to herd doc-writing cats through motivation and a pinch of automation </conf/eu/2017/speakers/#speaker-floriana-pagano>`_
+* Kathleen De Roo - `Manage your docs like an archivist </conf/eu/2017/speakers/#speaker-kathleen-de-roo>`_
+
+Important Ticket Pricing Update
+-------------------------------
+
+**The short version**: Based on community feedback, we reduced our ticket prices so that they are more accessible and inclusive to our European community, and refunded the difference to those who already purchased tickets.
+
+The new prices are:
+
+* 250 EUR for Corporate tickets (previously 300)
+* 150 EUR for Independent tickets (previously 200)
+* 75 EUR for Student & Unemployed tickets (previously 100)
+
+You can check out the detailed ticket pricing on our `Tickets <http://www.writethedocs.org/conf/eu/2017/tickets/>`_ page.
+
+**The long version**: Each year we examine our ticket pricing structure and determine the different tiers based on factors like production costs, attendee experience, and sponsorship funds offset. Our goal has always been to maintain a low price point and therefore a lower entry barrier to our events, so that documentarians from all walks of life can attend. As our community grows, our costs both inside and outside the conference grow too, and we have been gradually raising prices accordingly.
+
+In particular, we have changed how we budget our conferences to allow for paying some of our production staff and keeping a small sum to reinvest in operations outside of the conference, such as meetups and infrastructure. All this has added some overhead to the actual cost of the conference, but in hindsight, we should have maintained a more gradual increase in our ticket prices.
+
+After we received some candid feedback from the community, we re-examined the pricing structure and realized that we simply raised the prices too drastically this year. For example, the 2017 price for Independent tickets was twice as much as 2016. This is something we didn't catch at first, and doesn't reflect our values of accessibility and inclusion.
```

Comment:
After we received some candid feedback from the community, we re-examined the pricing structure and realized that we raised the prices disproportionately this year.